pkg/api/validate/dynamic/dynamic.go

```diff
@@ -581,34 +588,33 @@ func (dv *dynamic) validateVnetLocation(ctx context.Context, vnetr azure.Resourc
 	return nil
 }
 
-func (dv *dynamic) ValidateSubnets(ctx context.Context, oc *api.OpenShiftCluster, subnets []Subnet) error {
-	dv.log.Printf("validateSubnet")
```

Comment:
I think we rely on this message for debugging cluster operations. This log message has been dropped from the ValidateSubnets flow, we should likely add it back in the `ValidateSubnets` function below.

Reply:
Good catch.  I somehow accidentally deleted this.  Putting it back now.